lib/addons/xray/index.ts

```diff
@@ -0,0 +1,40 @@
+import { KubernetesManifest } from "@aws-cdk/aws-eks";
+import { ManagedPolicy } from "@aws-cdk/aws-iam";
+import { assertEC2NodeGroup } from "../../cluster-providers";
+import { ClusterAddOn, ClusterInfo } from "../../stacks/cluster-types";
+import { loadYaml, readYamlDocument } from "../../utils/yaml-utils";
+
+export class XrayAddOn implements ClusterAddOn {
```

Comment:
Doc bloc

Reply:
Will address

Reply:
Well, it appears not a single addon has it atm.  Why are you singling me out :))?

Reply:
done